features/step_definitions/repository_steps.rb

```diff
@@ -295,6 +295,10 @@
   expect(page.driver.status_code).to eq(404)
 end
 
+Then(/^I should see a div with class "(.*?)"$/) do |arg1|
+  page.all("div.#{arg1}")
```

Comment:
Does `page.all` fail if the element has not been found or it just returns an empty array?

In the lines just below you can a similar selector that we know for sure to fail when is necessary: `expect(page).to have_selector`